package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "gpii-grunt-lint-all",
+    "description": "A Grunt plugin to run all GPII lint checks.",
+    "version": "1.0.0",
+    "homepage": "https://github.com/GPII/gpii-grunt-lint-all",
+    "author": {
+        "name": "Tony Atkins",
+        "email": "tony@raisingthefloor.org"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/GPII/gpii-grunt-lint-all"
+    },
+    "bugs": {
+        "url": "https://issues.gpii.net/"
+    },
+    "license": "BSD-3-Clause",
+    "engines": {
+        "node": ">= 0.8.0"
+    },
+    "scripts": {
+        "test": "node tests/all-tests.js"
+    },
+    "dependencies": {
+        "chalk": "2.4.1",
+        "eslint-config-fluid": "1.3.0",
+        "eslint-plugin-markdown": "1.0.0-beta.6",
+        "fluid-eslint": "2.10.2",
+        "fluid-grunt-eslint": "18.1.2",
+        "fluid-grunt-json5lint": "1.0.0",
+        "gpii-grunt-mdjson-lint": "1.0.0-dev.20180629T080410Z.02b3951",
+        "grunt": "~0.4.5",
```

Comment:
Ancient, non-fixed grunt version - 1.0.3 will hopefully produce a few fewer warnings in install

Reply:
I think that was the grunt plugin init script's choice, it's fixed now.  I also updated the markdown linting plugin, which was way out of date.